backend/tests/functional_tests/conftest.py

```diff
@@ -64,3 +66,16 @@ def get_token_for_logged_in_user(role: str) -> str:
 
     with app.app_context():
         return sign_in_user(email, password)
+
+
+def create_tag_and_assign_to_category(key: str, name: str, scan_category_key: str) -> str:
```

Comment:
Please move this method somewhere else. Conftest is a place where we should store fixtures and methods that will be used by all tests. This function is specific, so it could be placed in a helper file or anywhere else.